tests/unit/plugins/LocalizeDateTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace ls\tests;
+
+/**
+ * Tests for the localize_date function.
+ */
+
+class LocalizeDateTest extends TestBaseClass
+{
+    public static function setupBeforeClass(): void
+    {
+        parent::setupBeforeClass();
+        \Yii::import('application.core.plugins.dateFunctions.EMFunctions', true);
```

Comment:
Import ? Not activate https://github.com/LimeSurvey/LimeSurvey/blob/4e5e6d2f3e2c17ce4bfdae9d86e4c9ef5eacc72a/tests/TestBaseClass.php#L147 ?

Reply:
Keep it but remove lss and importSurvey seems a better option.

Reply:
> Import ? Not activate

That's for calling the function inside the plugin directly.
Not related with LSS.

Reply:
Then : remove the lss file and the import no ?